 website2/_maincontent.scss

```diff
@@ -0,0 +1,223 @@
+.main-content {
+  display: flex;
+  margin-top: 1%;
+  margin-left: 10%;
+  margin-right: 10%;
+  // background-color:red;
+
+  .form-container {
+    width: 50%;
+    border:1px solid grey;
+
+   
+ // background-color:yellow;
+    .form-content {
+
+
+      &--row {
+        display: flex;
+        padding: 3%;
+        margin-left: 15%;
+        margin-right: 15%;
+
+        input, textarea {
+
+          margin-left: auto;
+
+        }
+        input{
+              height: 2rem;
+              width: 60%;
+              font-size: large;
+        }
+        textarea{
+          height: 4rem;
+          width: 60%;
+          font-size: small;
+        }
+       
+      }
+       .checkboxData{
+          display: flex;
+          margin-left: 24%;
+          margin-right: 30%;
+          font-size: small;
+          color: grey;
+          
+        }
+        .form-button{
+          display: flex;
+          justify-content: center;
+
+
+        
+        .reset-button{
+  background-color: #4CAF50; 
+    border: none;
+    color: white;
+    padding: 15px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+}
+.register-button{
+  background-color: #008CBA; /* Green */
+    border: none;
+    color: white;
+    padding: 15px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin-left: 10px;
+}
+        
+       
+         
+        
+      }
+    }
+  }
+
+  .output-container {
+    width: 40%;
+    margin-left: 8%;
+    border:1px solid grey;
+
+    #button143{
+    width:350px;
+    height: 50px;
+    
+  }
+  #button143:hover{
+    display: block;
+  }
```

Comment:
don't use id to style the element.
naming convention should be proper
use sass way to write the css.